bogota-july-2018/Dominik Ortiz Galop/css/normalize.css

```diff
@@ -0,0 +1,343 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Did you really need this ?

Reply:
Yes because normalize.css render all elements more consistently across browsers but Hmmmm... i don't think i need in this project because it's to small to normalize or reset style across browsers if was a bigger project maybe makes more sense